Dockerfile

```diff
@@ -20,6 +17,9 @@ COPY bin  /bin
 COPY tools/install_conf /tools/
 RUN /tools/install_conf
 
+RUN apt-get install -q -y binutils openjdk-11-jdk-headless
```

Comment:
..please don't do this! follow the existing way of doing stuff... and move it back where it was - install_conf is a more  lightweight finalizer step ; moving this behind it will cause that every time in that file changes "java" is reinstalled...

we can't just switch to jdk11 only - there should be a way to still support java8 (and optionally java11) at the same time... in tools/install_sdk.bash  (which you have just removed) there were an approach to switch between version...(but it turned out that sdkman sucks in a container environment like this - so I replaced it with a plain jdk8)

I'm not sure about moving JAVA_HOME ...it have been appended to the initialization file "/etc/profile.d/confs.sh" - we should either use that or not at all; right now it only has env variables...but moving just one is a bad idea